lib/dalli/protocol/meta.rb

```diff
@@ -85,6 +85,8 @@ def write_storage_req(mode, key, raw_value, ttl = nil, cas = nil, options = {})
                                         bitflags: bitflags, cas: cas,
                                         ttl: ttl, mode: mode, quiet: quiet?, base64: base64)
         write(req)
+        write(value)
+        write(TERMINATOR)
```

Comment:
@danmayer So it looks like we're just pulling the value out of the RequestFormatter's `meta_set` here, is that right?  If so, why are we leaving the value as a parameter to the `meta_set` method?

Reply:
Yup that is the change, good call I will remove the unused parameter.

Reply:
ah, so we still need to insert the size in the command `#{value.bytesize}"` so we can either leave it as a param as is... or we could pass a new param of `value_size` in these cases it doesn't seem to have any impact on the performance.

Reply:
Either way is fine.

Reply:
cool I will leave as is for now then

Reply:
any remaining concerns @petergoldstein or good to merge this?